maap/maap.py

```diff
@@ -19,6 +19,7 @@
 from maap.AWS import AWS
 from maap.dps.DpsHelper import DpsHelper
 from maap.utils import endpoints
+from maap.utils import job_utils
```

Comment:
minor: can we just name it `jobs` since already in the `utils` package? this would also make it consistent with `endpoints`